### PR TITLE
Event ICS button

### LINKED ISSUE #108: Events page: Add a .ics button per event?
Was discussed awhile ago as an improvement for events

### PR DESCRIPTION
**Pull request by garrying**
Fixes #108 

- [ ] Works on Chrome for Android (Yet to be tested)
- [x] Improve file naming: Consistent `event.ics`
- [x] Safari: FileSave.js is not working, and the downloaded file is just labeled `Unknown`
- [x] Works in iOS Safari
- [x] _Add to Google Calendar_ link

### CODE REVIEW

**Comment by dcwalk** [comment]
To make sure understand -- we expecting a per-event ics that people will download and add to an event.  I'm wondering if this could partially address #59 once the mobile is tested.

It looks good in the browser so far!

**Comment by garrying** [comment]
This could fix #59, but I'm a little weary about adapting this method of creating an ICS for cases where there are many events. Followed up with a possible server-side fix that needs to be tested.

As an additional feature to cover more cases, I wonder if it makes sense to have an Add to Google Calendar link that goes to gcal's _Add Event_ form with the fields prepopulated. Eventbrite uses this through their _Add to Calendar_ link on events.

**Comment by dcwalk** [comment]
Add to google calendar would be awesome!  Just dropping the eventbrite example you referenced because it didn't immediately come to mind: 
screen shot 2017-03-28 at 5 05 48 pm 1

File: _layouts/post.html [hunk 1]
```diff
@@ -31,6 +31,27 @@ <h1 class="post-title heading" itemprop="name headline">{{ page.title }}</h1>
...
+      <div class="event-add">
+        {% capture event_title %}{{ page.title }}{% endcapture %}
+        {% capture event_location %}{{ page.location }}{% endcapture %}
+        {% capture event_description %}{{ page.text|strip }} For details, link here: {{ page.url|strip | prepend: site.url }}{% endcapture %}
+        <script type="text/javascript">
+          var event_data = {
+            LOCATION:'{{ event_location }}',
+            SUMMARY:'{{ event_title }}',
+            DESCRIPTION:'{{ event_description }}',
+            DTSTART:'{{ page.date | date: "%-m/%-d/%Y" }} {{ page.startTime | date: "%-I:%M %P" }}',
```
Comment (benhylau) [inline _layouts/post.html]: This seems to already be in the format that would work for Android Calendar. Is that correct?

Comment (garrying) [reply _layouts/post.html]: This is a different format that `ics.js` expects. I'll see if I can get it to use the Android one—should probably just have one format for all this ICS stuff.

**Comment by dcwalk** [comment]
I'm sorry for never doing the final review :/ @garrying can you remind me where we are at with this?

**Comment by garrying** [comment]
@dcwalk My bad, I should have followed up. We left off at checking the new buttons on Android.

**Comment by dcwalk** [comment]
Okay! How about we make a plan to sort out the final steps on this at the work sesh on Jul 22?

**Comment by dcwalk** [comment]
We think this works! We'll merge in for now and reticket issues on android if they crop up!

**Pull request merged**